doc/mgmt/OpenConfig_PortChannel_Interface.md

```diff
@@ -0,0 +1,279 @@
+# OpenConfig support for PortChannel (aggregate) interface.
+
+# High Level Design Document
+#### Rev 0.1
+
+# Table of Contents
+  * [List of Tables](#list-of-tables)
+  * [Revision](#revision)
+  * [About This Manual](#about-this-manual)
+  * [Scope](#scope)
+  * [Definition/Abbreviation](#definitionabbreviation)
+  * [1 Feature Overview](#1-feature-overview)
+    * [1.1 Requirements](#11-requirements)
+      * [1.1.1 Functional Requirements](#111-functional-requirements)
+      * [1.1.2 Configuration and Management Requirements](#112-configuration-and-management-requirements)
+    * [1.2 Design Overview](#12-design-overview)
+      * [1.2.1 Basic Approach](#121-basic-approach)
+      * [1.2.2 Container](#122-container)
+  * [2 Functionality](#2-functionality)
+      * [2.1 Target Deployment Use Cases](#21-target-deployment-use-cases)
+  * [3 Design](#3-design)
+    * [3.1 Overview](#31-overview)
+    * [3.2 DB Changes](#32-db-changes)
+      * [3.2.1 CONFIG DB](#321-config-db)
+      * [3.2.2 APP DB](#322-app-db)
+      * [3.2.3 STATE DB](#323-state-db)
+      * [3.2.4 ASIC DB](#324-asic-db)
+      * [3.2.5 COUNTER DB](#325-counter-db)
+    * [3.3 User Interface](#33-user-interface)
+      * [3.3.1 REST API Support](#332-rest-api-support)
+      * [3.3.2 gNMI Support](#333-gnmi-support)
+  * [4 Flow Diagrams](#4-flow-diagrams)
+  * [5 Error Handling](#5-error-handling)
+  * [6 Unit Test Cases](#6-unit-test-cases)
+    * [6.1 Functional Test Cases](#61-functional-test-cases)
+    * [6.2 Negative Test Cases](#62-negative-test-cases)
+  
+# List of Tables
+[Table 1: Abbreviations](#table-1-abbreviations)
+[Table 2: OC YANG SONiC YANG Mapping](#4-flow-diagrams)
+
+# Revision
+| Rev |     Date    |       Author          | Change Description                |
+|:---:|:-----------:|:---------------------:|-----------------------------------|
+| 0.1 | 02/24/2024  | Satoru Shinohara | Initial version                   |
+
+# About this Manual
+This document provides general information about the OpenConfig configuration of portchannel (aggregate) interface in SONiC.
+
+# Scope
+- This document describes the high level design of OpenConfig configuration of portchannel interfaces via REST and gNMI in SONiC.
+- This does not cover the SONiC KLISH CLI.
+- This covers only the portchannel interfaces configuration.
+- This does not support subinterfaces configuration.
+- This does not cover gNMI subscription.
```

Comment:
Your code and UT already covers on-change subscription support.
e.g. `root@sonic:/# gnmi_cli -insecure -logtostderr  -address 100.94.113.103:8080 -query_type s -streaming_type ON_CHANGE -v 0 -target OC-YANG -q /openconfig-interfaces:interfaces/interface[name=PortChannel103]/config `

Reply:
Yes, will update the document with additional headings to show subscription support. Thank you for catching it.
Will mark as resolved once update is pushed.